src/main/java/persistence/sql/ddl/H2SqlTypeMapper.java

```diff
@@ -0,0 +1,23 @@
+package persistence.sql.ddl;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+
+public class H2SqlTypeMapper implements SqlTypeMapper {
+
+    Map<String, String> sqlType;
+
+    public H2SqlTypeMapper() {
+        sqlType = new HashMap<>();
+        sqlType.put("Long", "BIGINT");
+        sqlType.put("String", "VARCHAR(255)");
+        sqlType.put("Integer", "INT");
+    }
```

Comment:
자바의 타입을 문자열 대신 클래스를 사용해보면 타입 안정성을 가져갈 수 있을 것 같아요! :)

```suggestion
    Map<Class<?>, String> sqlType;

    public H2SqlTypeMapper() {
        sqlType = new HashMap<>();
        sqlType.put(Long.class, "BIGINT");
        sqlType.put(String.class, "VARCHAR(255)");
        sqlType.put(Integer.class, "INT");
    }
```